packages/squiggle-lang/src/fr/list.ts

```diff
@@ -357,12 +367,20 @@ export const library = [
     examples: [`List.slice([1,2,5,10],1,3)`],
     definitions: [
       makeDefinition(
-        [frArray(frGeneric("A")), frNumber, frOptional(frNumber)],
+        [
+          frArray(frGeneric("A")),
+          frNumber,
+          frNamed("index", frOptional(frNumber)),
```

Comment:
Wrong name, copy-pasted from `List.make`? Should be `end`, and `start` for the previous `frNumber`.

Reply:
(I'm removing this now, fixing in a new PR)